dist/translation-keys.txt

```diff
@@ -1,12 +1,12 @@
 Autoscale                                              // components/modebar/buttons.js:139
 Box Select                                             // components/modebar/buttons.js:103
-Click to enter Colorscale title                        // plots/plots.js:437
+Click to enter Colorscale title                        // plots/plots.js:338
```

Comment:
hmm, I should have just ignored the changes to this file, since they're only line number changes... I should revisit this so it doesn't get regenerated just because you run `test-syntax` locally.